image_topic: "/image_publisher_1702948963007019154/image_raw"
base_frame: "base_link"
tool_frame: "link_6"
save_path: "/tmp/test_data"
